### PR TITLE
OSDOCS-9277-new:State NMState wait-ip parameter in dual-stack network…

### PR DESCRIPTION
**Pull request by dfitzmau**
WILL NEED A CHANGE MANAGEMENT NOTIFICATION. After QE approve, I'll send out a CM for 4.12 to 4.15 changes.

Version(s):
4.12 to 4.16

Issue:
[OSDOCS-9277](https://issues.redhat.com/browse/OSDOCS-9277)

Link to docs preview:
[Optional: Deploying with dual-stack networking](https://75869--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#modifying-install-config-for-dual-stack-network_ipi-install-installation-workflow)

- [x] SME has approved this change.
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc [hunk 1]
```diff
@@ -28,6 +28,23 @@ serviceNetwork:
 - fd03::/112
 ----
 
+[IMPORTANT]
+====
+If you specified an NMState configuration in the `networkConfig` section of your your `install-config.yaml` file , ensure you add `interfaces.wait-ip: ipv4+ipv6` to the NMSate YAML file to resolve an issue that prevents your cluster from deploying on a dual-stack network.
+
+.Example NMSate YAML configuration file that includes the `wait-ip` parameter
```
Comment (cybertron) [inline modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc:35]: *NMState

**Review by cybertron — COMMENTED** [review]
One typo, but otherwise lgtm, thanks!

File: modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc [hunk 2]
```diff
@@ -28,6 +28,23 @@ serviceNetwork:
 - fd03::/112
 ----
 
+[IMPORTANT]
+====
+If you specified an NMState configuration in the `networkConfig` section of your your `install-config.yaml` file , ensure you add `interfaces.wait-ip: ipv4+ipv6` to the NMState YAML file to resolve an issue that prevents your cluster from deploying on a dual-stack network.
```
Comment (rbbratta) [inline modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc:33]: double `your`.

Comment (dfitzmau) [reply modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc]: Updated

**Pull request merged**